demonstrations/tutorial_guide_to_pennylane_knowing_qiskit.py

```diff
@@ -0,0 +1,239 @@
+# breaking changes were to be expected. Although learning a new language can be hard, PennyLane has very
+# simple tools that will help via the PennyLane-Qiskit plugin: your gateway to the land of PennyLane
+# that even lets you keep your existing Qiskit work without even having to know a ton about how PennyLane
+# works.
+#
+# There are two functions you need to know about:
+#
+# - :func:`~pennylane.from_qiskit`: converts an entire Qiskit ``QuantumCircuit`` — the whole thing — into
+#   PennyLane. It will faithfully convert Qiskit-side measurements (even mid-circuit measurements) or
+#   you can append PennyLane-side measurements directly to it.
+#
+# - :func:`~pennylane.from_qiskit_op`: converts a ``SparsePauliOp`` in Qiskit 1.0 to the equivalent operator in
+#   PennyLane.
+# 
+#
+# These two functions give you all that you need to access PennyLane’s features and user-interface
+# starting from the Qiskit side. As an example, let’s say you have the following code in Qiskit that
+# prepares a Bell state.
+#
+
+from qiskit import QuantumCircuit
+
+qc = QuantumCircuit(2)
+qc.h(0)
+qc.cx(0, 1)
+
+######################################################################
+# To convert this circuit to PennyLane, you just use ``qml.from_qiskit``:
+#
+
+import pennylane as qml
+
+pl_func = qml.from_qiskit(qc)
+print(qml.draw_mpl(pl_func, style="pennylane")())
+
+######################################################################
+# Want to measure some expectation values of Pauli operators, as well? Use ``qml.from_qiskit_op`` to
+# convert a ``SparsePauliOp`` into PennyLane’s equivalent operator.
+#
+
+from qiskit.quantum_info import SparsePauliOp
+
+qiskit_pauli_op = SparsePauliOp("XY")
+pl_pauli_op = qml.from_qiskit_op(qiskit_pauli_op)
+
+######################################################################
+# Then, you can *append* the expectation value measurement — done with ``qml.expval`` — to the PennyLane
+# circuit when you create it with ``qml.from_qiskit``:
+#
+
+pl_func = qml.from_qiskit(qc, measurements=[qml.expval(pl_pauli_op)])
+print(qml.draw_mpl(pl_func)())
+
+######################################################################
+# And just like that, you’re in PennyLane land! Now you might be asking: “What is ``pl_func`` and how
+# do I use it further?” To answer those questions, we need to get to know PennyLane a little better.
+#
+
+######################################################################
+# Get to Know PennyLane 🌞
+# ------------------------
+#
+# Let’s go back to that Qiskit circuit (``qc``) that we created earlier. If we want to execute that
+# circuit in Qiskit and get some results, we can do this:
+#
+
+from qiskit.primitives import StatevectorSampler
+
+qc.measure_all()
+
+sampler = StatevectorSampler()
+
+job_sampler = sampler.run([qc], shots=1024)
+result_sampler = job_sampler.result()[0].data.meas.get_counts() # or call get_counts()
```

Comment:
```suggestion
result_sampler = job_sampler.result()[0].data.meas.get_counts()
```
Minor mistake I think